paper-toast.html

```diff
@@ -192,7 +192,7 @@
 
         /**
          * Show the toast. Without arguments, this is the same as `open()` from `IronOverlayBehavior`.
-         * @param {Object|string|undefined} properties Properties to be set before opening the toast.
+         * @param {Object|string} [properties] Properties to be set before opening the toast.
```

Comment:
I think that the syntax for making a param optional is `{Object=|string=}` or maybe `{(Object|string)=}`

See: https://developers.google.com/closure/compiler/docs/js-for-compiler#types